src/iocore/net/YamlSNIConfig.cc

```diff
@@ -442,6 +452,12 @@ template <> struct convert<YamlSNIConfig::Item> {
     if (node[TS_client_sni_policy]) {
       item.client_sni_policy = node[TS_client_sni_policy].as<std::string>();
     }
+    if (node[TS_server_cipher_suite]) {
+      item.server_cipher_suite = node[TS_server_cipher_suite].as<std::string>();
+    }
+    if (node[TS_server_TLSv1_3_cipher_suites]) {
```

Comment:
Don't we need to check `TS_USE_TLS_SET_CIPHERSUITES`?

Reply:
Huh. That's a good point. I followed the lead of TS_USE_HELLO_CB for `TS_valid_tls_versions_in`, etc., which `#if` around the declarations, but they also don't check around the yaml nodes like this either.

Reply:
Updated. Thanks for catching this.

Reply:
I thought this would cause a compile error but I was wrong. `TSDECL` declares the name regardless of the availability, so we can have this without the check.

Reply:
Ah, right. It's declared, just not in the `valid_sni_config_keys` list. OK. Taking the checks back out.